src/gui/gui.ts

```diff
@@ -559,6 +574,26 @@ To automatically trust this fingerprint for future connections, click Yes. To re
         <source>Run Wizard</source>
         <translation type="unfinished"></translation>
     </message>
+    <message>
+        <location filename="res/MainWindowBase.ui" line="435"/>
+        <source>S&amp;ave configuration</source>
```

Comment:
Should there be an ampersand here?

Reply:
This comes from MainWindowBase.ui. Looks like ampersand has a special meaning in Qt.